volumina/utility/signalingDefaultDict.py

```diff
@@ -20,16 +19,15 @@ class SignalingDefaultDict( QObject ):
     """
     updated = pyqtSignal(object) # set(updated_keys) 
     
-    def __init__(self, parent, default_factory, *args, **kwargs):
-        assert default_factory is not None, "You must provide a default_factory."
+    def __init__(self, parent, default_value):
         super( SignalingDefaultDict, self ).__init__(parent)
-        self._dict = defaultdict(default_factory, *args, **kwargs)
+        self._dict = {}
+        self.default_value = default_value
+        assert not callable(default_value), "Pass a default_value, not a default_factory"
```

Comment:
We should leave it with callable.
Currently, factory function is used incorrectly it always returns the same object of `QPen`.
Which may lead to accidental changes to the default value,  it's fixable with callable, but not with a default value.
```python
p = QPen()
a = SignalingDefaultDict(None, p)
p1 = a['mypen']
p2 = a['myanotherpen']
p1.setColor(Qt.Red) # will change p2
```
also `defaultdict` will materialize requested keys, not sure how important this is